reactive/containerd.py

```diff
@@ -248,6 +269,50 @@ def merge_custom_registries(config_directory, custom_registries,
     return registries
 
 
+def invalid_custom_registries(custom_registries):
+    """
+    Validate custom registries from config.
+
+    :param str custom_registries: juju config for custom registries
+    :return: error string for blocked status if condition exists, None otherwise
+    :rtype: Optional[str]
+    """
+    try:
+        registries = _registries_list(custom_registries)
+    except json.JSONDecodeError:
+        return "Failed to decode json string"
+    except ValueError as e:
+        return "custom_registries is not a list"
```

Comment:
ValueError is pretty generic and could mean a lot of things. I don't think it's safe to assume that a ValueError here is always the "not a list" case. Consider creating a new exception with a name like InvalidCustomRegistries where "not a list" or "JSON decode error" are possible messages that it has.